src/services/ChallengeService.js

```diff
@@ -284,10 +287,26 @@ searchChallenges.schema = {
     updatedDateEnd: Joi.date(),
     createdBy: Joi.string(),
     updatedBy: Joi.string(),
-    memberId: Joi.number().integer().positive()
+    memberId: Joi.number().integer().positive(),
+    sortBy: Joi.string()
   })
 }
 
+
+async function validateSortQueryFields (criteria) {
```

Comment:
why didn't you just use `Joi` for validation? The valid values should be in the app-constants.js and you should use the `.valid()` method from Joi for validation.

Reply:
I saw a function which validated fields in similar way hence to maintain consistency I did this. will change it as per your comment

Reply:
@ThomasKranitsas change has been pushed. Kindly review